course-structure.json

```diff
@@ -1,37 +1,75 @@
 [
     {
-        "title": "Client Interaction with the Solana Network",
```

Comment:
Thinking about this, what if we create a new grouping for the "Getting Started" and "Cryptography Fundamentals" as opposed to adding them to the existing first module? Going to approve and merge for now but might update later.

Reply:
## Re: cryptography fundamentals in the first module

My thinking here was a lot of the content from 'cryptography fundamentals' originally came from 'intro to reading data'. See https://github.com/Unboxed-Software/solana-course/pull/221/files#diff-1aa79c99b26d7434dcc4a6f1046a3fc61c2b441837be5e337ff531bd5e14a4cdL3

 - since handling keypairs is a prerequisite to reading data, and using keypairs to read data is a natural flow on from understanding keypairs, these two fit together
 - The current set up where student do practical things with Solana in the first module is good, and we should keep that.

## Re: getting started in the first module

This could go elsewhere. It just needed to be in *a* module to appear in the contents, and the top level item in `course-structure` is an array of modules. We could put it in it's own, single lesson module if you prefer? WDYT?

Reply:
Tweaked it to make 'Getting Started' it's own module, it now looks like this:

![Screenshot 2023-09-12 at 13 19 22](https://github.com/Unboxed-Software/solana-course/assets/172594/ffb2911d-8d84-45c4-bcfb-91b31242a13b)